language/Portuguese_Brazil/strings.po

```diff
@@ -2509,146 +2515,154 @@ msgid ""
+msgstr "  %2$s %5$d.%6$03d kHz\n"
 
 #: src/frontend/mame/ui/devopt.cpp:85
 msgid "* Video:\n"
-msgstr ""
+msgstr "* Vídeo:\n"
 
 #: src/frontend/mame/ui/devopt.cpp:90
 #, c-format
 msgid "  Screen '%1$s': Vector\n"
-msgstr ""
+msgstr "  Tela '%1$s': Vetor\n"
 
 #: src/frontend/mame/ui/devopt.cpp:99
 #, c-format
 msgid "  Screen '%1$s': %2$d × %3$d (V) %4$f Hz\n"
-msgstr ""
+msgstr "  Tela '%1$s': %2$d × %3$d (V) %4$f Hz\n"
 
 #: src/frontend/mame/ui/devopt.cpp:100
 #, c-format
 msgid "  Screen '%1$s': %2$d × %3$d (H) %4$f Hz\n"
-msgstr ""
+msgstr "  Tela '%1$s': %2$d × %3$d (H) %4$f Hz\n"
 
 #: src/frontend/mame/ui/devopt.cpp:113
 msgid "* Sound:\n"
-msgstr ""
+msgstr "* Som:\n"
 
 #: src/frontend/mame/ui/devopt.cpp:133
 #, c-format
 msgid "  %1$d×%2$s\n"
-msgstr ""
+msgstr "  %1$d×%2$s\n"
 
 #: src/frontend/mame/ui/devopt.cpp:134
 msgid "  %2$s\n"
-msgstr ""
+msgstr "  %2$s\n"
 
 #: src/frontend/mame/ui/devopt.cpp:164
 #, c-format
 msgid ""
 "* BIOS settings:\n"
 "  %1$d options    [default: %2$s]\n"
 msgstr ""
+"* Configurações da BIOS:\n"
+"  %1$d opções    [padrão: %2$s]\n"
 
 #: src/frontend/mame/ui/devopt.cpp:204 src/frontend/mame/ui/devopt.cpp:220
 #: src/frontend/mame/ui/devopt.cpp:265
 #, c-format
 msgid "  %1$s    [default: %2$s]\n"
-msgstr ""
+msgstr "  %1$s    [padrão: %2$s]\n"
 
 #: src/frontend/mame/ui/devopt.cpp:209 src/frontend/mame/ui/devopt.cpp:225
 #, c-format
 msgid "  %1$s\n"
-msgstr ""
+msgstr "  %1$s\n"
 
 #: src/frontend/mame/ui/devopt.cpp:230
 msgid "* DIP switch settings:\n"
-msgstr ""
+msgstr "* Configurações das chaves DIP:\n"
 
 #: src/frontend/mame/ui/devopt.cpp:232
 msgid "* Configuration settings:\n"
-msgstr ""
+msgstr "* Definição das configurações:\n"
 
 #: src/frontend/mame/ui/devopt.cpp:234
 msgid "* Input device(s):\n"
-msgstr ""
+msgstr "* Dispositivo(s) de entrada:\n"
 
 #: src/frontend/mame/ui/devopt.cpp:236
 #, c-format
 msgid "  User inputs    [%1$d inputs]\n"
-msgstr ""
+msgstr "  Entradas do usuário    [%1$d entradas]\n"
 
 #: src/frontend/mame/ui/devopt.cpp:238
 #, c-format
 msgid "  Mahjong inputs    [%1$d inputs]\n"
-msgstr ""
+msgstr " Entradas Mahjong    [%1$d entradas]\n"
 
 #: src/frontend/mame/ui/devopt.cpp:240
 #, c-format
 msgid "  Hanafuda inputs    [%1$d inputs]\n"
-msgstr ""
+msgstr "  Entrada Hanafuda    [%1$d entradas]\n"
 
 #: src/frontend/mame/ui/devopt.cpp:242
 #, c-format
 msgid "  Gambling inputs    [%1$d inputs]\n"
-msgstr ""
+msgstr "  Entradas para jogos de apostas    [%1$d entradas]\n"
 
 #: src/frontend/mame/ui/devopt.cpp:244
 #, c-format
 msgid "  Analog inputs    [%1$d inputs]\n"
-msgstr ""
+msgstr "  Entradas Analógicas    [%1$d entradas]\n"
 
 #: src/frontend/mame/ui/devopt.cpp:246
 #, c-format
 msgid "  Adjuster inputs    [%1$d inputs]\n"
-msgstr ""
+msgstr "  Entradas Adjuster    [%1$d entradas]\n"
 
 #: src/frontend/mame/ui/devopt.cpp:248
 #, c-format
 msgid "  Keypad inputs    [%1$d inputs]\n"
-msgstr ""
+msgstr "  Entradas de teclado numérico   [%1$d entradas]\n"
```

Comment:
"Keypad" in this case isn't necessarily numeric.  The difference between "keypad" and "keyboard" is just that the presence of "keyboard" inputs causes full keyboard mode to be enabled, while "keypad" inputs doesn't.  Is "telcado numérico" the best translation?

Reply:
It can be "teclado" also.